docs/docs/ref/rows_cols.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'columns/1'
+package: ALS Library
+group: Formatting
+predicates:
+- {sig: 'columns', args: {
+    1: 'Output ListOfRows as a minimal columnar array of terms',
+    2: 'Like columns/1, outputting to a given stream',
+    3: 'Output ListOfRows as a columnized rectangular array of terms'
+  }}
+---
+## FORMS
+
+`columns(ListOfRows)`
+
+`columns(ListOfRows, OutputStream)`
+
+`columns(ListOfRows, ColWidths, OutputStream)`
+
+## DESCRIPTION
+
+**`columns/1`** If ListOfRows is a list of lists of terms (Rows), and  
```

Comment:
That’s because they exactly match the source description lines in rows_cols.pro.
Aesthetically, I’m ok with it, especially ‘cause anything else is going to get into auto or manual line length adjustments/typsetting.  Note that on
        https://cols--alsprolog.netlify.com/docs/ref/rows_cols.html
the description lines are roughly the same length as everything else, from top down to footer.

Reply:
With the latest commit, the problem has gotten worse, with one sentence jutting out (see screenshot below). Similarly, when the text is zoomed on a narrow viewport, the wrapping becomes very ragged (2nd screenshot).

Maintaining these manually line-broken paragraphs will be a hassle when the text is displayed in different layouts, font-sizes, etc. Markdown paragraphs are automatically line-wrapped (when rendered), so they are naturally "layout-agnostic". Wouldn't it be easier to let Markdown wrap most paragraph text for us?

![cols--alsprolog netlify com_docs_ref_rows_cols html](https://user-images.githubusercontent.com/1627217/58507496-e24bb780-815f-11e9-8926-cb21a03c9c72.png)

![image](https://user-images.githubusercontent.com/1627217/58587191-be08dd00-822a-11e9-8367-39c8f5109ce8.png)

Reply:
Keeping in mind that:
a.  these lines are mechanically extracted from the source in library/rows_cols.pro;
b.  one does want the source comment in library/rows_cols.pro to look reasonable, as I believe is does now;
c.  part of the goal of the library (and builtins) doc mechanism is to make it easier to end up with documented code (cf. e.g., builtins/filepath.pro and others); it’s better to have some slightly rough-looking doc instead of no doc at all; but certainly as good as is reasonably possible.
Can you make a technical suggestion to maintain a,b&c, and to achieve getting the mechanical extraction to accomplish: "let Markdown wrap most paragraph text” ??

Reply:
Latest commit resolved this, and we can continue discussing doc authoring issues via email.